configs/mattors/fba/baseline.py

```diff
@@ -0,0 +1,175 @@
+custom_imports = dict(
+    imports=['mmedit.core.optimizer.radam'], allow_failed_imports=False)
```

Comment:
Unnecessary imports

Reply:
Ok, I will delete it. I learned this from some issue in other OpenMMLab repo.